frontend/smart-kickers-game/src/components/Game/GameStatistics/NumberOfShots.js

```diff
@@ -4,9 +4,9 @@ import './GameStatistics.css';
 function NumberOfShots({ statistics }) {
   return (
     <>
-      <div className="table-item">{statistics?.blueShotsCount}</div>
+      <div className="table-item">{statistics ? statistics.blueShotsCount : '0'}</div>
```

Comment:
In theory `statistics` should be here every time now, but `blueShotsCount` probably not.

Reply:
Changed to statistics?.blueShotsCount and same for whiteShotsCount.